src/app/(app)/missing-shoes/add/components/MissingShoesActions.tsx

```diff
@@ -0,0 +1,75 @@
+"use client";
+
+import React, { useMemo } from "react";
+import ActionsMenu from "@/common/ActionMenu/ActionsMenu";
+import {
+  Trash2 as TrashIcon,
+  Edit as EditIcon,
+  CheckCircle as CheckCircleIcon,
+  AlertCircle as AlertCircleIcon,
+} from "lucide-react";
+import { MissingShoeStatus } from "@/constant/constant";
+
+export const MissingShoesActions = ({
+  missingShoesId,
+}: {
+  missingShoesId: number;
+}) => {
+  const handleEditInfo = (id: number) => {
+    return id;
+  };
+
+  const handleDeleteRequest = async (id: number) => {
+    return id;
+  };
+
+  const handleLeaveRequestStatus = async (
+    id: number,
+    status: MissingShoeStatus
+  ) => {
+    return {
+      id,
+      status,
+    };
+  };
+
+  const actionMenu = useMemo(
+    () => [
+      {
+        title: "Found",
+        id: 4,
+        onClick: () => {
+          handleLeaveRequestStatus(missingShoesId, MissingShoeStatus.Found);
```

Comment:
rename function

Reply:
@muhammad-bilal-ali-khokhar would be great if you suggest a better name too.

Reply:
@nadirhussain786  I suggest you `handleMissingShoesStatus`